### PR TITLE
Update ndisplay title

### PR DESCRIPTION
**Pull request by dstansby**
# Description
Depends on https://github.com/napari/napari/pull/6192

## Type of change

- [x] Fixes or improves existing content
- [ ] Adds new content page(s)
- [ ] Fixes or improves workflow, documentation build or deployment

# References

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added [alt text](https://webaim.org/techniques/alttext/) to new images included in this PR

### CODE REVIEW

File: docs/tutorials/fundamentals/viewer.md [hunk 1]
```diff
@@ -340,7 +340,7 @@ The console (when available) appears at the bottom of the viewer as shown below:
 
 +++
 
-#### Toggle ndisplay (2D/3D) button
+#### Toggle 2D/3D button
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/viewer.md:343]:
```suggestion
#### Toggle 2D/3D view
```
not sure we need button? Not a blocker.

**Review by psobolewskiPhD — APPROVED** [review]
Thanks, made a tiny suggestion, also took the liberty to update the PR description.

**Comment by Czaki** [comment]
I have merged https://github.com/napari/napari/pull/6192. Could you resolve the pending discussion?

File: docs/tutorials/fundamentals/viewer.md (hunk 1 above)
Comment (dstansby) [reply docs/tutorials/fundamentals/viewer.md:343]: 👎  to doing this in isolation in this PR, since many other headings use "button" at the moment.

**Comment by dstansby** [comment]
> Could you resolve the pending discussion?

👍 done

**Pull request merged**